.gitattributes

```diff
@@ -1,4 +1,4 @@
-*                  text=auto
+*                  text=auto eol=lf
```

Comment:
Is this necessary? I believe `*   text=auto` already normalizes newlines the way `eol=lf` does (ref. https://git-scm.com/docs/gitattributes#_end_of_line_conversion)

Reply:
Yes, it's necessary. Windows guys often commit their CRLF characters to the open-source world by mistake. Their editors all use CRLF by default. As I know, some people even have `eol=crlf` and `autocrlf=true` in their global config.

We don't use Windows-related tools so we can promise we don't need any CRLF in all files. It's OK to set all files to `eol=lf`.

Reply:
I'm not disagreeing we should normalize line endings and prevent CRLF characters from ending up in this project @LitoMore, I'm asking whether your change actually makes sense as, from the documentation, it would seem to me that `* text=auto` already normalizes newlines to LF. From the reference I provided:

> If you want to ensure that text files that any contributor introduces to the repository have their line endings normalized, you can set the text attribute to "auto" for all files.
>
> ```
> *	text=auto
> ```

Reply:
OK. I reverted this change.